### ISSUE TITLE
datagrid filter button missing aria-label

### ISSUE THREAD

**Issue by rotj** [vmware-clarity/ng-clarity#1785]
## Describe the bug

The datagrid filter button added with [clrDgField] is missing aria-label. It was present in v16. For example, the "Name" column would have "Toggle Name filter" as the aria-label on the filter button.

## How to reproduce

See https://clarity.design/documentation/datagrid/code/binding-properties

Compare to https://v16.clarity.design/documentation/datagrid/code/binding-properties

## Expected behavior

All filter buttons have an aria-label based on column name.

## Versions

**Clarity version:**

v17.x

**Framework version:**
Angular 19

**Comment by dtsanevmw** [comment]
Hello,

This issue should be fixed in our latest version v17.11.0. Thanks for reporting!

Regards,
Clarity Team
